### ISSUE TITLE
REPL: jump to function edit deprecation warning

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#26058]
```julia
julia> methods(hash)
julia> 52^Q
WARNING: Base.edit is deprecated: it has been moved to the standard library package `InteractiveUtils`.
Add `using InteractiveUtils` to your imports.
 in module Base
WARNING: Base.edit is deprecated: it has been moved to the standard library package `InteractiveUtils`.
Add `using InteractiveUtils` to your imports.
 in module REPL
in #59 at /Users/jameson/julia/usr/share/julia/site/v0.7/REPL/src/REPL.jl
```

**Comment by JeffBezanson** [comment]
Is this a recent commit? Works for me.

**Comment by fredrikekre** [comment]
Fixed by #25978 I think.
